packages/react-devtools-shared/src/devtools/views/SuspenseTab/SuspenseScrubber.css

```diff
@@ -0,0 +1,60 @@
+.SuspenseScrubber {
+  position: relative;
+  width: 100%;
+  height: 1.5rem;
+  border-radius: 0.75rem;
+  padding: 0.25rem;
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+}
+
+.SuspenseScrubber:has(.SuspenseScrubberInput:focus-visible) {
+  outline: 2px solid var(--color-button-background-focus);
+}
+
+.SuspenseScrubberInput {
+  position: absolute;
+  width: 100%;
+  opacity: 0;
+  height: 0px;
+  overflow: hidden;
+}
+
+.SuspenseScrubberInput:focus {
+  outline: none;
+}
+
+.SuspenseScrubberStep {
+  cursor: pointer;
+  flex: 1;
+  height: 100%;
+  padding-right: 1px; /* we use this instead of flex gap to make every pixel clickable */
+  display: flex;
+  align-items: center;
+}
+.SuspenseScrubberStep:last-child {
+  padding-right: 0;
+}
+
+.SuspenseScrubberBead, .SuspenseScrubberBeadSelected {
+  flex: 1;
+  height: 0.5rem;
+  background: var(--color-background-selected);
+  border-radius: 0.5rem;
+  background: var(--color-selected-tree-highlight-active);
+  transition: all 0.3s ease-in-out;
+}
+
+.SuspenseScrubberBeadSelected {
+  height: 1rem;
+  background: var(--color-background-selected);
+}
+
+.SuspenseScrubberBeadSelected:hover {
+  height: 0.75rem;
+}
+
+.SuspenseScrubberBead:hover {
+  height: 0.75rem;
+}
```

Comment:
These should shrink when you hover over the parent. Otherwise you get a jitter when you just barely hover over the outer rim that then retracts due to hover which means it's no longer hovered. That loop goes on forever

https://github.com/user-attachments/assets/525fbfc7-7442-4393-8af0-424afdfc551e

The larger the default font size, the more likely you get this scenario.

It felt actually counter-intuitive that it shrinks on hover. I expected expansion on hover

Reply:
Hm. We should make the line-height fixed there. I'll make it fixed height. I can also make it overflow hidden because the idea is that it should not pop out of the container that's the hit target. The hit target should be the outer box, not the thing animating. In fact, with a view transition it wouldn't even be a hit target.

It starts moving in the direction that it would if you clicked it. It gives a nice squeezing effect when it's in the middle so when you cut it off it cuts off the "flow" there.

Reply:
Oh it was because the `:hover` was on the bead. It should be on the step like the listeners are.